components/content-blocks/SummitStatus/index.js

```diff
@@ -1,17 +1,41 @@
 import PropTypes from "prop-types";
+import { useState } from "react";
 import { Container } from "@rubin-epo/epo-react-lib";
+import { useTranslation } from "react-i18next";
+import NestedContext from "@/contexts/Nested";
+import UnitLocalization from "@/components/layout/UnitLocalization";
 import WidgetGrid from "@/components/layout/WidgetGrid";
 import WidgetPreview from "@/components/layout/WidgetPreview";
 
-const SummitStatus = ({ summitStatusLayout, widgetPreviews = [] }) => (
-  <Container bgColor="neutral95" width="wide" paddingSize="small">
-    <WidgetGrid>
-      {widgetPreviews.map(({}, i) => (
-        <WidgetPreview key={i}></WidgetPreview>
-      ))}
-    </WidgetGrid>
-  </Container>
-);
+const SummitStatus = ({ summitStatusLayout, widgetPreviews = [] }) => {
+  /** this logic should be changed to useRouter after i18n refactor */
+  const {
+    i18n: { language = "en" },
+  } = useTranslation();
+  const [windspeedUnit, setWindspeedUnit] = useState(
+    language === "en" ? "kn" : "m/s"
```

Comment:
setting the English default as knots since imo it is the standard imperial unit for windspeed, but maybe mph makes more sense since our users probably aren't sailors or pilots.